reactComponents/src/searchbar/index.tsx

```diff
@@ -3,27 +3,27 @@ import React, { FC } from 'react';
 import './App.css';
 import { Provider } from 'react-redux';
 import { store } from './redux/store';
-import TokenSearch from "./tokenSearch";
+import TokenSearch from './tokenSearch';
 import { RenderProps } from '../types';
-import config from './config'
+import config from './config';
 
 export const SearchBar: FC<RenderProps> = (renderProps: RenderProps) => {
-  return (  
-      <Provider store={store}>      
-      {
-        !config.IS_ENV_PRODUCTION &&        
-        <TokenSearch 
-          customWrapper={renderProps?.customWrapper}
-          customSearchInput={renderProps?.customSearchInput}
-          customSearchFilter={renderProps?.customSearchFilter}
-          customLoading={renderProps?.customLoading}
-          customChip={renderProps?.customChip}
-          customResult={renderProps?.customResult}
-          customTokenDetail={renderProps?.customTokenDetail}        
-          customActions={renderProps?.customActions}        
-          customAllChip={renderProps?.customAllChip}
+  return (
+    <Provider store={store}>
+      {!config.IS_ENV_PRODUCTION && (
```

Comment:
whats this ENV in here for? I recall taking this out?

How are we supposed to flag this to show when the ENV to control it is the same ENV that needs to be flagged in the RT?

Why is this library package using an ENV at all? Why not send it through as props - otherwise, how is the user of this component supposed to know/track when ENV vars they need to set on the main application in order to control the functionality of a dependency?

I thought we had previously talked and agreed that dependencies like this should not rely on ENV vars

Reply:
ahhh I see, you hardcoded the config - I would suggest removing this IS_ENV_PRODUCTION